procurement_service_timesheet/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Procurement service timesheet",
+    "version": "10.0.1.0.0",
+    "category": "Procurements",
+    "website": "https://github.com/OCA/manufacture/"
+               "tree/10.0/procurement_service_timesheet",
+    "author": "Simone Rubino, Agile Business Group, "
+              "Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "auto-install": True,
```

Comment:
Is this intentional?

Reply:
Yes this is intentional: this module restores the standard behaviour (creating a task) expected by the user when selling a service configured with 'create task'.